test/e2e/utils_test.go

```diff
@@ -170,96 +167,26 @@ func getTestManifest(t *testing.T, baseManifestPath string) (io.Reader, error) {
 		}
 	}
 
-	t.Logf("using modified %v manifest", baseManifestPath)
-
-	// patch kong image.
-	return manifestsReader, nil
-}
-
-const imageKustomizationContents = `resources:
-- base.yaml
-images:
-- name: kong/kubernetes-ingress-controller
-  newName: %v
-  newTag: '%v'
-`
-
-// patchControllerImage takes a manifest, image, and tag and runs kustomize to replace the
-// kong/kubernetes-ingress-controller image with the provided image. It returns the location of kustomize's output.
-func patchControllerImage(baseManifestReader io.Reader, image string, tag string) (io.Reader, error) {
-	workDir, err := os.MkdirTemp("", "kictest.")
+	manifestsReader, err = patchControllerStartTimeout(manifestsReader, 120, time.Second*3)
 	if err != nil {
-		return nil, err
-	}
-	defer os.RemoveAll(workDir)
-	orig, err := io.ReadAll(baseManifestReader)
-	if err != nil {
-		return nil, err
-	}
-	err = os.WriteFile(filepath.Join(workDir, "base.yaml"), orig, 0o600)
-	if err != nil {
-		return nil, err
-	}
-	kustomization := []byte(fmt.Sprintf(imageKustomizationContents, image, tag))
-	err = os.WriteFile(filepath.Join(workDir, "kustomization.yaml"), kustomization, 0o600)
-	if err != nil {
-		return nil, err
+		t.Logf("failed patching controller timeouts (%v), using default manifest %v", err, baseManifestPath)
+		return manifestsReader, nil
 	}
 
-	kustomized, err := kustomizeManifest(workDir)
+	manifestsReader, err = patchLivenessProbes(manifestsReader, 0, 10, time.Second*15, time.Second*3)
 	if err != nil {
-		return nil, err
+		t.Logf("failed patching kong liveness (%v), using default manifest %v", err, baseManifestPath)
+		return manifestsReader, nil
 	}
-	return bytes.NewReader(kustomized), nil
-}
 
-const kongImageKustomizationContents = `resources:
-- base.yaml
-images:
-- name: kong
-  newName: %v
-  newTag: '%v'
-- name: kong/kong-gateway
-  newName: %v
-  newTag: '%v'
-`
-
-func patchKongImage(baseManifestsReader io.Reader, image string, tag string) (io.Reader, error) {
-	workDir, err := os.MkdirTemp("", "kictest.")
+	manifestsReader, err = patchLivenessProbes(manifestsReader, 1, 15, 3, 10)
```

Comment:
yikes, this one would wait 3 and 10 nanoseconds 😬 

```suggestion
	manifestsReader, err = patchLivenessProbes(manifestsReader, 1, 15, time.Second*3, time.Second*10)
```